sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -278,183 +441,298 @@ private JobManifestTasks getJobManifestTasks(TextAnalyticsActions actions) {
         return jobManifestTasks;
     }
 
-    private List<EntitiesTask> toEntitiesTask(TextAnalyticsActions actions) {
+    private EntitiesLROTask toEntitiesLROTask(RecognizeEntitiesAction action) {
+        if (action == null) {
+            return null;
+        }
+        final EntitiesLROTask task = new EntitiesLROTask();
+        task.setParameters(getEntitiesTaskParameters(action)).setTaskName(action.getActionName());
+        return task;
+    }
+
+    private List<EntitiesTask> toEntitiesTasks(TextAnalyticsActions actions) {
         final List<EntitiesTask> entitiesTasks = new ArrayList<>();
         for (RecognizeEntitiesAction action : actions.getRecognizeEntitiesActions()) {
-            if (action == null) {
-                entitiesTasks.add(null);
-            } else {
-                entitiesTasks.add(
-                    new EntitiesTask()
+            entitiesTasks.add(
+                action == null
+                    ? null
+                    : new EntitiesTask()
                         .setTaskName(action.getActionName())
-                        .setParameters(
-                            new EntitiesTaskParameters()
-                                .setModelVersion(action.getModelVersion())
-                                .setLoggingOptOut(action.isServiceLogsDisabled())
-                                .setStringIndexType(StringIndexType.UTF16CODE_UNIT)));
-            }
+                        .setParameters(getEntitiesTaskParameters(action)));
         }
         return entitiesTasks;
     }
 
-    private List<PiiTask> toPiiTask(TextAnalyticsActions actions) {
+    private EntitiesTaskParameters getEntitiesTaskParameters(RecognizeEntitiesAction action) {
+        return (EntitiesTaskParameters) new EntitiesTaskParameters()
+                                         .setStringIndexType(StringIndexType.UTF16CODE_UNIT)
```

Comment:
has a user always not been able to override the stringIndexType default?

Reply:
Yes. The StringIndexType is default the UTF16CODE_UNIT until StringIndexType is on. Currently. Java still not yet make the StringIndexType publicly